### PR TITLE
Make patientSearchWidget responsive

### PR DESCRIPTION
**Pull request by brandones**
supports https://github.com/openmrs/openmrs-module-registrationapp/pull/81

https://pihemr.atlassian.net/browse/UHM-4635

Screenshot from 2020-06-23 19-02-52
Screenshot from 2020-06-23 19-03-09
Screenshot from 2020-06-23 19-03-30
Screenshot from 2020-06-23 19-03-53
Screenshot from 2020-06-23 19-04-08

### CODE REVIEW

File: omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp [hunk 1]
```diff
@@ -13,28 +13,27 @@
         listableAttributeTypes.push('${ ui.encodeHtml(it) }');
     <% } %>
     var lastViewedPatients = [];
-    <%  if (showLastViewedPatients && !doInitialSearch) {
```
Comment (brandones) [inline omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp:16]: Just fixed the indentation. Use "Hide whitespace changes"

File: omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ function PatientSearchWidget(configuration){
         attributeHeaders = attributeHeaders.concat('<th>'+value+'</th>');
     });
     var tableId = 'patient-search-results-table';
-    var tableHtml = '<table class="table table-sm table-responsive-sm table-responsive-md table-responsive-lg table-responsive-xl}" id="'+tableId+'">'+
+    var tableHtml = '<table class="table table-sm" id="'+tableId+'">'+
```
Comment (brandones) [inline omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js:20]: My #1 hack for making things responsive is to try deleting most of the code

Comment (HerbertYiga) [reply omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js:20]: @brandones  hi brandon,well done on this,on your last screens short   ie (Find patient) i see the a drop down menu when the screen is full,i was thinking we could have a drop down menu on only minimizing the screen,what do you think?

File: README.md [hunk 3]
```diff
@@ -8,7 +8,14 @@ OpenMRS Core Apps Module
...
 
-# Dashboard widgets development
+# Development
+
+As usual, you can just use `mvn openmrs-sdk:watch` for development.
+
+However, in order to see changes to `scss` files, you'll also need to run
+`mvn process-resources -Pwatch-sass`.
+
+## Dashboard Widgets
```
Comment (mogoodrich) [inline README.md:18]: Thanks for improving the docs!

File: omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp [hunk 4]
```diff
@@ -110,15 +115,18 @@
     </div>
 </div>
 <div class="row">
-    <div class="col-md-12 col-sm-12 col-xs-12">
+    <div class="col-12">
```
Comment (mogoodrich) [inline omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp:118]: Just for my knowledge, this doesn't actually change anything, but just removes unnecessary classes, correct?

File: omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js (hunk 2 above)
Comment (brandones) [reply omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js:20]: Hi Herbert, this PR doesn't relate to the nav bar.

File: omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp (hunk 4 above)
Comment (brandones) [reply omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp:118]: Exactly. Also `col-xs-12` is BS3, `col-12` is BS4.

**Pull request merged**